explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -77,6 +77,13 @@ async function getRecentTransactions(
     pageNum?: number
 ): Promise<TxnData[]> {
     try {
+        if (totalTx === 0) {
```

Comment:
Abusing  the meaning of`totalTx` here is a bit confusing to the reader, I would move line 81 - 85 to line 262 with the condition that `pagedNum === 1`

Reply:
Done